pkg/crc/preflight/preflight.go

```diff
@@ -56,7 +56,7 @@ func (check *PreflightCheck) shouldWarn() bool {
 
 func (check *PreflightCheck) doCheck() error {
 	if check.checkDescription == "" {
-		// warning, this is a programming error
+		panic("Should not happen, empty description for check")
```

Comment:
Shouldn't we also tell again which `configKeySuffix` description is empty so it will provide bit more context when hit the panic? 

```
panic(fmt.Sprintf("Should not happen, empty description for check %s", check.configKeySuffix ))
```